### ISSUE TITLE
Provide <Plug> mappings for use in Undotree_CustomMap

### ISSUE THREAD

**Issue by roryokane** [mbbill/undotree#52]
I want to swap the meanings of `<` and `>` when browsing the undo-tree, because it feels more natural to me that `<` would go back in time. However, I am currently unable to set that up, despite the existence of `Undotree_CustomMap`:

```vim
function g:Undotree_CustomMap()
    " the below leads to infinite recursion (E223)
    nmap <buffer> < >
    nmap <buffer> > <
    " the below just uses the default behavior of < and >
    nnoremap <buffer> < >
    nnoremap <buffer> > <
    " the below works, but sets a global option, messing up
    "  the mappings in all other buffers
    set langmap=<>,><
endfunction
```

This could be solved by adding `<Plug>` mappings for all of the supported commands in the undo-tree browser, which could be used like this:

```vim
function g:Undotree_CustomMap()
    nmap <buffer> < <Plug>UndotreePreviousSavedState
    nmap <buffer> > <Plug>UndotreeNextSavedState
endfunction
```

**Comment by mbbill** [comment]
It's fixed in the latest patch fa018f38252f58073f2987f8bf0d2d4a61e07277.
I've also swapped the default key bindings for < and >, actually it's a bug.

**Comment by roryokane** [comment]
Thank you. Fixing the default keybindings was better than I had hoped for.
